apps/emqx/i18n/emqx_schema_i18n.conf

```
@@ -1747,6 +1747,110 @@ server_ssl_opts_schema_gc_after_handshake {
     }
 }
 
+server_ssl_opts_schema_enable_ocsp_stapling {
+    desc {
+        en: "Whether to enable OCSP stapling for the listener.  If set to true,"
+            " requires defining the OCSP responder URL and issuer PEM path."
+        zh: "是否为监听器启用OCSP装订功能。 如果设置为 true，"
+            "需要定义OCSP响应者URL和发行者PEM路径。"
+    }
+    label: {
+        en: "Enable OCSP Stapling"
+        zh: "启用OCSP订书机"
+    }
+}
+
+server_ssl_opts_schema_ocsp_responder_url {
+    desc {
+        en: "URL for the OCSP responder to check the server certificate against."
+        zh: "用于检查服务器证书的OCSP响应器的URL。"
+    }
+    label: {
+        en: "OCSP Responder URL"
+        zh: "OCSP响应者URL"
+    }
+}
+
+server_ssl_opts_schema_ocsp_issuer_pem {
+    desc {
+        en: "PEM-encoded certificate of the OCSP issuer for the server certificate."
+        zh: "服务器证书的OCSP签发者的PEM编码证书。"
+    }
+    label: {
+        en: "OCSP Issuer Certificate"
+        zh: "OCSP发行人证书"
+    }
+}
+
+server_ssl_opts_schema_ocsp_refresh_interval {
+    desc {
+        en: "The period to refresh the OCSP response for the server."
+        zh: "为服务器刷新OCSP响应的周期。"
+    }
+    label: {
+        en: "OCSP Refresh Interval"
+        zh: "OCSP刷新间隔"
+    }
+}
+
+server_ssl_opts_schema_ocsp_refresh_http_timeout {
+    desc {
+        en: "The timeout for the HTTP request when checking OCSP responses."
+        zh: "检查OCSP响应时，HTTP请求的超时。"
+    }
+    label: {
+        en: "OCSP Refresh HTTP Timeout"
+        zh: "OCSP刷新HTTP超时"
+    }
+}
+
+server_ssl_opts_schema_enable_crl_check {
+    desc {
+        en: "Whether to enable CRL verification and caching for this listener."
+        zh: "是否为该听众启用CRL验证和缓存。"
+    }
+    label: {
+        en: "Enable CRL Check"
+        zh: "启用CRL检查"
+    }
+}
+
+server_ssl_opts_schema_crl_cache_urls {
+    desc {
+        en: "URLs for CRL servers to fetch and cache CRLs from."
```

```suggestion
        en: "URLs for Certificate Revocation List (CRL) servers to fetch and cache CRLs from."
```
I think it would be nice to explain the acronyms at least once.